doc/community-release-build.md

```diff
@@ -0,0 +1,202 @@
+# 3scale-Operator Community Release
+
+[Introduction](#introduction)  
+[Main Steps Briefly](#main-steps-briefly)  
+[Versioning Strategy](#versioning-strategy)  
+[Steps to Perform Community Release in 3scale-operator repo](#steps-to-perform-community-release-in-3scale-operator-repo)  
+[Prepare release in community-operators-prod repo](#prepare-release-in-community-operators-prod-repo)
+[References](#references)
+
+
+## Introduction
+* Community Release created in parallel with Minor Product Releases or patch releases, and not for Major release.
```

Comment:
@valerymo  not for Major - is this correct. 
There will most likely never be another major 3scale release but if there was I think we would also release major community

Reply:
@briangallagher  Thank you for comments. Updated, separate commit. Hope now it looks better. Thanks